flow-report/src/sidebar/flow.tsx

```diff
@@ -0,0 +1,77 @@
+import {FunctionComponent} from 'preact';
+import {classNames, useCurrentLhr, useFlowResult} from '../util';
+
+const FlowStepIcon: FunctionComponent<{mode: LH.Gatherer.GatherMode}> = ({mode}) => {
+  return <div className={`FlowStepIcon FlowStepIcon--${mode}`}></div>;
+};
+
+const SidebarFlowStep: FunctionComponent<{
+  mode: LH.Gatherer.GatherMode,
+  href: string,
+  label: string,
+  hideTopLine: boolean,
+  hideBottomLine: boolean,
+  isCurrent: boolean,
+}> = ({href, label, mode, hideTopLine, hideBottomLine, isCurrent}) => {
+  return (
+    <a
+      className={classNames('SidebarFlowStep', {'Sidebar--current': isCurrent})}
```

Comment:
seems like `Sidebar--current` should be `SidebarFlowStep--current`?

Reply:
`Sidebar--current` is also used for `SidebarSummary`